tests/test_writer.py

```diff
@@ -22,7 +22,7 @@ def test_invalid_args(self, local_remote_dir: Tuple[str, str]):
         local, _ = local_remote_dir
         dataset = SequenceDataset(100)
         columns = dict(zip(dataset.column_names, dataset.column_encodings))
-        with pytest.raises(ValueError, match=f'.*Invalid Writer argument.*'):
+        with pytest.raises(ValueError):
```

Comment:
Is there any specific reason for removing `match`? Valuing against the exception message would be good to ensure we capture the correct exception.

Reply:
Fixed